components/pipedream/actions/create-subscription/create-subscription.mjs

```diff
@@ -0,0 +1,34 @@
+import pipedream from "../../pipedream.app.mjs";
+
+export default {
+  key: "pipedream-create-subscription",
+  name: "Creates a Subscription",
+  description: "Create a Subscription. [See Doc](https://pipedream.com/docs/api/rest/#subscriptions)",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    pipedream,
+    emitterId: {
+      propDefinition: [
+        pipedream,
+        "emitterId",
+      ],
+    },
+    eventName: {
```

Comment:
This prop is reused in another component, so I think it could be moved to the app file